backend/src/plugins/Mutes/commands/MutesCmd.ts

```diff
@@ -27,10 +27,17 @@ export const MutesCmd = mutesCmd({
     let totalMutes = 0;
     let hasFilters = false;
 
-    let stopCollectionFn;
-    let stopCollectionTimeout;
+    let stopCollectionFn = () => {
+      return;
+    };
```

Comment:
```suggestion
    let stopCollectionFn = () => {};
```

Reply:
This would return an error because of an empty block.
I would prefer to do this instead of telling tslint to ignore it.

Reply:
What about `() => void 0;`?